### DISCUSSION TITLE
Expand Batch Attachment Upload path tables

### DISCUSSION THREAD

**Discussion by bronwyncombs** [specify/specify7#5984] [Feature Requests]
**Is your feature request related to a problem? Please describe.**

**From Dan B.:** 

> Hi Specify,
> We had a request from a user to upload 1000s of files to the preparation table, looking at the documentation for the Sp7 functions it doesn't look like that table is in the list of mapping options. Is there a setting to make more table links available, or would this be an enhancement request?

**Reported By**
CSIRO

**Additional context**

Current tables/paths available: 

<img width="279" alt="Screenshot 2024-08-27 at 8 43 12 AM" src="https://github.com/user-attachments/assets/dc60dffa-71f0-44c0-846d-554195351bbb">

I assume this would be a unique barcode field, or possibly require an ID to ensure attaching to the correct prep on a CO.

**Comment by bronwyncombs** [comment]
Dan confirmed CSIRO requests for the `barCode` field

**Comment by grantfitzsimmons** [comment]
Barcode would be great for RBGE and Philippe Verley at IRD

Requested by: Philippe Verley at IRD on the [Speciforum](https://discourse.specifysoftware.org/t/extend-batch-attachment-uploader/2040)

**Comment by grantfitzsimmons** [comment]
>Is it about time to just make it generic, with using any field from any table that supports attachments? Trivial for fields. It's a bit involved to span relationships (to-manys are a no-go. to-ones, maybe --- but at that point, might as well support in workbench).

@realVinayak
